pkg/server/translation.go

```diff
@@ -248,6 +253,13 @@ func (t *TranslateServiceServer) UpdateTranslation(
 		}
 	}
 
+	if err = model.UpdateTranslation(
```

Comment:
Could you add an integration test where a message for an original translation is updated/appended ? 
So that we know everything is working as intended.